Automatic Validation ended with:
> 2024-04-23T11:10:21.7026258Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:26.7644353Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/17/17.0.11.10/BellSoft.LibericaJDK.17.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/17/17.0.11.10/BellSoft.LibericaJDK.17.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 17 (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-i586.msi
-InstallerSha256: 0C2FFFDECA2B8D91EB46BC66C59895C4F3656A4B00B10F90C379928662C30DDD
-ProductCode: '{CCEF6701-30B7-4CDD-A23B-3E4AA4723BC2}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 17 (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-i586.msi
+  InstallerSha256: 0C2FFFDECA2B8D91EB46BC66C59895C4F3656A4B00B10F90C379928662C30DDD
+  ProductCode: '{CCEF6701-30B7-4CDD-A23B-3E4AA4723BC2}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 17 (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-aarch64.msi
   InstallerSha256: 06C45277E64B201827C75DB583413EC125E4440B2F837AB603DBBC1A6CA38801

